apps/console/src/app/(app)/(org)/[slug]/[workspaceName]/(manage)/sources/new/_components/sentry-source-item.tsx

```
@@ -0,0 +1,282 @@
+												</button>
+											))}
+										</div>
+									)}
+								</div>
+
+								<div className="text-center text-sm text-muted-foreground">
+									Missing a project?{" "}
+									<button
+										onClick={handleConnect}
+										className="text-blue-500 hover:text-blue-600 underline-offset-4 hover:underline transition-colors"
+									>
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)